data/json/monsters/feral_humans.json

```
@@ -690,5 +690,29 @@
     "zombify_into": "mon_zombie_soldier",
     "anger_triggers": [ "FRIEND_DIED", "FRIEND_ATTACKED", "HURT" ],
     "flags": [ "SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "HUMAN", "CAN_OPEN_DOORS", "PATH_AVOID_DANGER_1" ]
+  },
+  {
+    "id": "mon_feral_soldier_extra",
+    "looks_like": "mon_feral_soldier",
+    "copy-from": "mon_feral_soldier",
+    "description": "Wearing a military uniform with an armored vest, this person seems to be a highly trained military professional, now turned feral.  Tense, dirty hands grip a handgun and combat knife as their bloodshot eyes scan for threats.",
+    "starting_ammo": { "9mm": 3 },
+    "special_attacks": [
+      {
+        "type": "gun",
+        "cooldown": 10,
+        "move_cost": 50,
+        "//": "(movecost is 50 to show that the feral prepper doesn't take time to aim, and just shoots barbaric-ly)",
```

```suggestion
        "//": "(movecost is 50 to show that the feral solider doesn't take time to aim, and just shoots barbaric-ly)",
```
I'm super guilty of leaving comments in my copy-paste but it probably should be slightly corrected at least.